js/components/closed-caption-multi-audio-menu/closedCaptionMultiAudioMenu.js

```diff
@@ -0,0 +1,128 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+var Utils = require('../utils');
+var classnames = require('classnames');
+var CONSTANTS = require('../../constants/constants');
+var Tab = require('./tab');
+var MultiAudioTab = require('./multiAudioTab');
+
+var ClosedCaptionMultiAudioMenu = React.createClass({
+  componentDidMount: function() {
+    var multiAudioCol = ReactDOM.findDOMNode(this.refs.multiAudioCol);
+    var closeCaptionsCol = ReactDOM.findDOMNode(this.refs.closeCaptionsCol);
+    if (multiAudioCol && closeCaptionsCol && typeof Utils.getMaxElementWidth === 'function') {
```

Comment:
I don't think we need to check `Utils.getMaxElementWidth`, we should be able to trust that our own functions are defined.

Reply:
Thank you for the review! Good point, I think we can remove that check.

Reply:
\+